tools/integration_tests/util/creds_tests/testdata/create_service_account.sh

```diff
@@ -11,12 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# Create service account if does not exist.
 
 SERVICE_ACCOUNT=$1
-SERVICE_ACCOUNT_ID=$2
-# Delete service account if already exist.
-gcloud iam service-accounts delete $SERVICE_ACCOUNT_ID
+
+gcloud iam service-accounts create $SERVICE_ACCOUNT --description="$SERVICE_ACCOUNT" --display-name="$SERVICE_ACCOUNT" &>> output.txt
 if [ $? -eq 1 ]; then
-  echo "Service account does not exist."
+  if grep "already exists within project" output.txt; then
```

Comment:
I'm not sure I understand the logic here. `already exists within project` sounds like it's a case where we don't want to do anything further.. But why are we exiting with 1 (error) in the else case (not already existing account) ?

Reply:
I think we're exiting with 1 when `gcloud iam service-accounts create` fails with some other error apart from when the service account already exists within project.

Reply:
Yes, if we get any other error we will exit with exit code 1

Reply:
Okay. Makes sense.